docs/dunst.5.pod

```diff
@@ -100,6 +100,12 @@ the notification on the left border of the screen while a horizontal offset of
 
 =back
 
+=item B<scale> (default: 0)
+
+Specifies a scale factor for dimensions to adapt notifications to the
+screen DPI. This does not apply to text. If 0 is specified, the scale
```

Comment:
Screen dpi or wayland scale factor

Reply:
That's the source of auto-detection, not really the purpose. I have pushed another tentative to not mention DPI. Otherwise, please propose the whole text.

Reply:
It's documentation for the dpi scaling at the same time :)
The text should also be scaled with the scale factor. If that's not the case then something is wrong.

I would suggest this as text:
```
Specifies a scale factor for dimensions to adapt notifications to HiDPI
screens. This scales the notification geometry and it's contents.
It is not recommended to use a fractional scaling factor, as this may result in
things being one pixel off. Try to use a whole number scaling factor and adjust
the font size and other sizes as needed. If 0 is specified, the scale factor is
auto-detected.
```

Reply:
Unfortunately, on X11, it does not scale text. Text is handled using Xft/DPI (from XSETTINGS, with a fallback to X resource database). I don't think this is very important, unless you use the `scale` settings, in this case, you don't control the whole scale.

Reply:
Weird I thought we controlled text size directly. The scale variable is kind of useless on X11 then. On wayland it also doesn't work currently. It doesn't scale the window dimensions, so the notification is cut off.

Reply:
We could make it an X11 only setting. Wayland does not really need it because there is only one way to get the scaling, so we are unlikely to be wrong. And since it does not scale text, we can just have a 3-state setting: integer-scaling, integer-scaling++, fractional-scaling. I am not good with names.

Reply:
Yeah but if the text size doesn't change from this scale variable it's pretty much useless. Is there a way to draw larger text independent from the X dpi?

Reply:
I don't know. But if there is, I wouldn't expect Dunst to override Xft/DPI (or Xft.DPI). Text scaling is pretty much the only thing that's easy to do on Linux without much surprise.